packages/drawer/README.md

```diff
@@ -49,12 +49,12 @@ function ExampleComponent() {
 
 ## Properties
 
-| Prop                    | Type        | Description                                        | Default |
-| ----------------------- | ----------- | -------------------------------------------------- | ------- |
-| `children` _(optional)_ | `ReactNode` | Children that will be rendered inside the `Drawer` |         |
-| `open`                  | `boolean`   | Determines if the `Drawer` is open or closed       | `false` |
-| `setOpen`               | `function`  | Callback to change the open state of the `Drawer`  |         |
-| `title`                 | `ReactNode` | Title of the `Drawer`                              |         |
+| Prop                    | Type                                         | Description                                                                                                     | Default |
+| ----------------------- | -------------------------------------------- | --------------------------------------------------------------------------------------------------------------- | ------- |
+| `children` _(optional)_ | `React.ReactNode`                            | Children that will be rendered inside the `Drawer`                                                              |         |
+| `onClose` _(optional)_  | `React.MouseEventHandler<HTMLButtonElement>` | Event handler called on close button click. If provided, a close button will be rendered in the `Drawer` header |         |
```

Comment:
Would you close the drawer by manually toggling the `open` prop after some action without a close button?